iaf-commons/src/main/java/nl/nn/adapterframework/util/StringUtil.java

```diff
@@ -220,4 +220,24 @@ public static int countRegex(String string, String regex) {
 			count++;
 		return count;
 	}
+
+	// Not sure what's better here...
+	// return input.substring(0, 1).toLowerCase() + input.substring(1);
```

Comment:
I don't think that it makes much of a difference here, what is better. I would perhaps slightly prefer this form but the code is short and low level, and both feel kind of ugly to me in different ways.